types/index.d.ts

```diff
@@ -93,44 +107,71 @@ declare namespace unified {
     /**
      * Transform a syntax tree by applying plugins to it.
      *
-     * @param node
-     * @param file `VFile` or anything which can be given to `vfile()`
-     * @param done Invoked when transformation is complete.
-     * Either invoked with an error or a syntax tree and a file.
+     * @param node Node to tranform
      * @returns `Promise` if `done` is not given. Rejected with an error, or resolved with the resulting syntax tree.
      */
     run(node: Node): Promise<Node>
+
+    /**
+     * Transform a syntax tree by applying plugins to it.
+     *
+     * @param node Node to tranform
+     * @param file `VFile` or anything which can be given to `vfile()`
+     * @returns `Promise` if `done` is not given. Rejected with an error, or resolved with the resulting syntax tree.
+     */
     run(node: Node, file: VFileCompatible): Promise<Node>
+
+    /**
+     * Transform a syntax tree by applying plugins to it.
+     *
+     * @param node Node to tranform
+     * @param done Invoked when transformation is complete.
+     */
     run(node: Node, done: RunCallback): void
+
+    /**
+     * Transform a syntax tree by applying plugins to it.
+     *
+     * @param node Node to tranform
+     * @param file `VFile` or anything which can be given to `vfile()`
+     * @param done Invoked when transformation is complete.
+     * Either invoked with an error or a syntax tree and a file.
+     * @returns `Promise` if `done` is not given. Rejected with an error, or resolved with the resulting syntax tree.
```

Comment:
It returns void in this case, should we keep description for `@returns`?

Reply:
good catch, removed